src/Cake.Web/Cake.Web.csproj

```diff
@@ -267,7 +267,53 @@
     <Content Include="Views\Blog\Category.cshtml" />
     <Content Include="Views\Addin\Index.cshtml" />
     <Content Include="Views\Shared\_AddinWarning.cshtml" />
-    <Content Include="App_Data\**\*.md" />
+    <Content Include="App_Data\blog\2015-07-05-cake-v0.4.3-released.md" />
```

Comment:
@patriksvensson looks like Visual Studio trying to be "clever". Do you want to put this back the way it was, or do you want them explicitly added into the csproj file?